packages/backend/src/accounting/psql/service.ts

```diff
@@ -160,3 +227,34 @@ async function postAccountWithdrawal(
 ): Promise<void | TransferError> {
   throw new Error('Not implemented')
 }
+
+async function getAccountBalances(
+  deps: ServiceDependencies,
+  account: LedgerAccount
+): Promise<AccountBalance> {
+  const transfers = await deps.ledgerTransferService.getAccountTransfers(
+    account.id
+  )
+
+  const credits = transfers.filter(
+    (transfer) => transfer.creditAccountId === account.id
+  )
+  const debits = transfers.filter(
+    (transfer) => transfer.debitAccountId === account.id
+  )
```

Comment:
This might work with a single `reduce` instead of two `filter`s
https://stackoverflow.com/a/67333661/12196980
What do you think about having `getAccountTransfers` do the split instead (returning `{ credits: LedgerTransfer[], debits: LedgerTransfer[] }`)